mealpy/mealpy.py

```diff
@@ -28,6 +28,35 @@
 COOKIES_FILENAME = 'cookies.txt'
 
 
+def get_cities():
```

Comment:
What's the advantage of extracting these out? IMO even if they do not require any authentication / state, having all API calls related to Mealpal within `MealPal` class would be more coherent. Any network calls to retrieve data from Mealpal is automatically associated with the `MealPal` class. There is no requirement to login on initialization of a `MealPal` object anyway.

Reply:
It's been hard for me to justify the existence of the `MealPal` class and to wrap my head around it. It has common smells of being a class for the sake of "everything is a class" from the Java world (debatable if it's non-idiomatic), imo. "What does a `MealPal` instance do?" is not immediately clear to me.

Did you intend this to be a client library? Maybe that's the perspective I did not initially see it as.

Reply:
Yes, you could view it from that perspective where the `MealPal` class is some library and the cli/script is just using the functionality in that library.

Reply:
FYI, I'm planning to move the "main()" and cli logic to a separate module, keep `mealpy.py` more focused.